### PR TITLE
feat(opentrons-ai-client): add code copy button

### PR DESCRIPTION
**Pull request by koji**
# Overview
add code copy button

https://github.com/Opentrons/opentrons/assets/474225/9aa16b52-b29e-4010-bc63-b8e9ffb9ee34

copy code button might be changed since the button hides the text.
~~Need to get feedback from product team.~~

align the button with Gradio style button
https://huggingface.co/spaces/Opentrons/OpentronsAI-v0.0.5

close AUTH-389

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

**Comment by koji** [comment]
- [ ] add test cases

**Review by ncdiehl11 — COMMENTED** [review]
code looks good, how can I test? I spun up the environment with `make -C opentrons-ai-client dev` but submitting a prompt doesn't seem to get a response

File: opentrons-ai-client/src/molecules/ChatDisplay/index.tsx [hunk 1]
```diff
@@ -52,41 +72,55 @@ export function ChatDisplay({ chat }: ChatDisplayProps): JSX.Element {
...
         >
           {content}
-        </Markdown> */}
-        <Markdown>{content}</Markdown>
+        </Markdown>
+        {role === 'assistant' ? (
+          <PrimaryButton
+            position={POSITION_ABSOLUTE}
+            right={SPACING.spacing16}
+            bottom={`-${SPACING.spacing16}`}
```
Comment (jerader) [inline opentrons-ai-client/src/molecules/ChatDisplay/index.tsx:80]: is there an alternative to using a negative bottom spacing here? Like adding a top spacing?

**Review by jerader — APPROVED** [review]
the video you provided lgtm and the code lgtm. I'm having similar issues as Nick and can't get a response from the ai when running only `make -C opentrons-ai-client dev`

**Pull request merged**